### DISCUSSION TITLE
Semaphore support group level permission

### DISCUSSION THREAD

**Discussion by venkatasankhya** [semaphoreui/semaphore#981] [Ideas]
is it possible to add AD group to the semaphore project team? 

right now, we can add only individual user to give permission to the semaphore project.

Ideally if we can add group, we won't need to add individual user to the project team.

**Comment by venkatasankhya** [comment]
@fiftin Please help me on this request

**Reply by tboerger** [reply]
Currently there is no support for group mapping integration but contributions for this are highly appreciated.
